script.js

```diff
@@ -1 +1,9 @@
 "use strict";
+document.getElementById('header_burger').onclick = function() {
+    document.getElementById('header_burger').classList.toggle('active');
```

Comment:
This is already 2nd time you have queried DOM for header_burger, please declare it globally so JS has to find it only once.

Reply:
> This is already 2nd time you have queried DOM for header_burger, please declare it globally so JS has to find it only once.

Added global variables instead queries.